### ISSUE TITLE
bazel build of tensorflow fails

### ISSUE THREAD

**Issue by pmiddend** [NixOS/nixpkgs#42809]
## Issue description

I just did a `nix-shell -p python36Packages.tensowflow` and got this error:

```
Loading:
Loading: 0 packages loaded
Analyzing: target //tensorflow/tools/pip_package:build_pip_package (2 packages loaded)
Analyzing: target //tensorflow/tools/pip_package:build_pip_package (29 packages loaded)
Analyzing: target //tensorflow/tools/pip_package:build_pip_package (123 packages loaded)
ERROR: /build/output/external/jpeg/BUILD:122:12: Illegal ambiguous match on configurable attribute "deps" in @jpeg//:jpeg:
@jpeg//:k8
@jpeg//:armeabi-v7a
Multiple matches are not allowed unless one is unambiguously more specialized.
ERROR: Analysis of target '//tensorflow/tools/pip_package:build_pip_package' failed; build aborted:
```

## Technical details

 - system: `"x86_64-linux"`
 - host os: `Linux 4.14.51, NixOS, 18.09pre143771.a8c71037e04 (Jellyfish)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.0.4`
 - channels(philipp): `"nixpkgs-18.09pre140731.c29d2fde74d"`
 - channels(root): `"nixos-18.09pre143771.a8c71037e04"`
 - nixpkgs: `/home/philipp/.nix-defexpr/channels/nixpkgs`

**Comment by twhitehead** [comment]
@abbradar @FRidh @jyp looks like [upstream breakage](https://github.com/tensorflow/tensorflow/issues/4807) due to tensorflow having used the value of `android_cpu` instead of `cpu` and bazel having changed the default value of `android_cpu`.

From what I've read, I believe you have to either downgrade bazel to 0.11, upgrade tensorflow, or manually [incorporate the `android_cpu` to `cpu` fix](https://github.com/tensorflow/tensorflow/pull/17508) into tensorflow

**Comment by jyp** [comment]
@pmiddend Master now uses a wheel-based build. So the title should be changed to "bazel build of tensorflow fails".

**Comment by pmiddend** [comment]
@jyp: fixed.

**Comment by yorickvP** [comment]
Fixed by #64716
